app/helpers/comment_mailer_helper.rb

```diff
@@ -0,0 +1,5 @@
+module CommentMailerHelper
+  def self.get_project_users(project)
+    ( project.project_owner.send_notifications? ? [project.project_owner] : [] ) | User.project_notifiable(project.id).to_a
```

Comment:
I refactored this a bit to use a scope and got rid of the local variable which is kind of a general goal of our methods at sport ngin. Local variables are a last resort